iai_naive_object_db/Object[] objects
---
uint8 ERROR_CODE
